lib/storage/client.js

```diff
@@ -1182,6 +1183,64 @@ class Client extends CryptoConsumer {
     })
     return listGroups
   }
-}
 
+  /**
+   * addGroupMembers adds members to the TozStore Group given.
+   *
+   * @param {string} groupId  UUID assigned by TozStore, used to identify a group.
+   * @param {GroupMember[]} groupMembers wraps together the ClientID, Capability and Membership Key for each client being added to the group
+   *
+   *
+   * @returns {GroupMember[]} A response from TozStore; the group members who have been added.
+   *
+   */
+  async addGroupMembers(groupId, groupMembers) {
+    if (!(groupMembers instanceof GroupMember)) {
```

Comment:
I applaud the effort, but its javascript so better to embrace its untyped nature and not sporadically type check things (e.g. we're not type checking the groupId but we are for group members). Also I think this will fail / doesn't lined up with the documentation as the docs and variable name indicate it can be more than one group member, in which case instanceof == Array and that would cause this method to fail.